### DISCUSSION TITLE
Inúteu… #3

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#531] [Curiosidades]
Post de referência: https://www.vish.blog.br/curiosidades/inuteu-3/

**Comment by the-coffee-monk** [comment]
Essa série *Inúteu* evoluiu incrivelmente desde a primeira "edição" e mesmo que às vezes eu não entenda direito alguns tópicos, ela é sensacional. :star\_struck:

**Comment by angellisy** [comment]
> [The Coffee Monk](</perfil/?user=the-coffee-monk>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/curiosidades/inuteu-3/#comment_1173882841>):

> Essa série *Inúteu* evoluiu incrivelmente desde a primeira "edição" e mesmo que às vezes eu não entenda direito alguns tópicos, ela é sensacional. 🤩

Não poderia ter dito melhor

**Comment by najarin-the-loremaster** [comment]
> [The Coffee Monk](</perfil/?user=the-coffee-monk>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/curiosidades/inuteu-3/#comment_1173882841>):

> Essa série *Inúteu* evoluiu incrivelmente desde a primeira "edição" e mesmo que às vezes eu não entenda direito alguns tópicos, ela é sensacional. 🤩

No começo eu fiquei um pouco apreensivo se o pessoal desse site gostaria então eu comecei devagar, com curiosidades não tão interessantes e, apesar de não ter havido lá muito engajamento, também não houve rejeição. Tem horas que alguns assuntos também me confundem, mas tudo bem, faz parte do processo.
